content/news/2025-09-24-bevy-0.17/index.md

```diff
@@ -60,13 +60,16 @@ Additionally, if you have a NVIDIA GPU, you can enable DLSS Ray Reconstruction w
 
 ### How it works
 
-TODO: Showcase the different aspects/steps of Solari (direct initial+temporal+spatial, indirect initial+temporal+spatial, world cache, DLSS RR)
+Our current implementation uses rayraced dissue direct and indirect lighting (also known as global illuminance), which are sampled via ReSTIR variants and uses a world-space irradiance cache to improve quality.
+Like all raytracing techniques, this produces results that are too noisy for realtime applications.
+To resolve this, you need to add a denoising step, which is currently handled via DLSS Ray Reconstruction, although we'd be happy to add support for alternate methods as well in the future.
```

Comment:
Idk if we want to commit to alternate denoisers. Like yes it's something I want, particularly FSR's DLSS-RR equivilant when they release it, but it's not something I'm activley looking at.

Reply:
I tried to frame this as "please, feel free to PR this!" not "we're going to do this". Open to other wording suggestions though!